test/basic_test.go

```diff
@@ -0,0 +1,49 @@
+package test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTerraformBasicExample(t *testing.T) {
+	t.Parallel()
+
+	awsProfile := os.Getenv("AWS_PROFILE")
+	require.NotEmpty(t, awsProfile)
+
+	awsRegion := os.Getenv("AWS_DEFAULT_REGION")
+	require.NotEmpty(t, awsRegion)
+
+	vpcName := os.Getenv("SIMPLE_EKS_TEST_VPC_NAME")
+	require.NotEmpty(t, vpcName)
+
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: "../examples/basic",
+		Vars: map[string]interface{}{
+			"aws_profile": awsProfile,
+			"aws_region":  awsRegion,
+			"vpc_name":    vpcName,
+		},
+		NoColor: true,
+	})
+
+	defer terraform.Destroy(t, terraformOptions)
+
+	terraform.InitAndApply(t, terraformOptions)
+
+	assert.Regexp(t,
```

Comment:
All of these asserts check values for resources created by the cluster. We should do assertions on resources created by the nodes.